package.json

```diff
@@ -21,7 +21,7 @@
     "test": "tape test/**/*.spec.js | tap-spec"
   },
   "name": "kubeutils",
-  "version": "1.8.5",
+  "version": "1.8.6",
```

Comment:
try to keep the version bumps separate. So develop the code, then in a single commit update the version in the changelog and package.json